TombEngine/Game/control/box.cpp

```diff
@@ -1124,13 +1125,7 @@ int StalkBox(ItemInfo* item, ItemInfo* enemy, int boxNumber)
 		return false;
 
 	int enemyQuad = (enemy->Pose.Orientation.y / ANGLE(90.0f)) + 2;
-	
-	int boxQuad;
-	if (z > 0)
-		boxQuad = (x > 0) ? 2 : 1;
-	else
-		boxQuad = (x > 0) ? 3 : 0;
-
+	int boxQuad = z <= 0 ? (x <= 0 ? 0 : 3) : (x > 0) + 1;
```

Comment:
Why this if-else statement was converted to almost unreadable ternary operation?

Reply:
TOMB4 :x

Reply:
Fixed